docs/apis/webhooks/lists/create-subscription.md

```diff
@@ -83,9 +83,15 @@ Content-Type: application/json
 
 ## URL validation
 
-Before a new subscription is created, SharePoint sends a request with a validation token in the body of the request to the service URL provided. Your service must respond to this request by returning the validation token.
+**IMPORTANT - Before a new subscription is created, SharePoint sends a request with a validation token in the querystring of the request to the notification URL provided. Your service must respond to this request by returning the validation token. If your service fails to validate the request in this way, the subscription is not created.**

```

Comment:
Please change this to use the MSDocs format for an important callout. it shoudl be a block quote in the format of:

```markdown
> [!IMPORTANT]
> markdown text goes here
```